README.md

```diff
@@ -60,16 +43,21 @@ Feel free to check the [issues page](../../issues/).
 
 ## Show your support
 
-Give a ⭐️ if you like this project!
+⭐️⭐️⭐️⭐️⭐
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
-
+- Microverse
+- A big thanks to Cindy Shin in Behance - (https://www.behance.net/adagio07)
+- Photo acknowledgements:
+- "https://www.vecteezy.com/free-vector/lines"- Lines Vectors by Vecteezy
+- "https://www.freepik.com/free-vector/black-carbon-fiber-texture-background_6918300.htm#query=carbon%20fibre&position=0&from_view=keyword" - Image by starline on Freepik
+- "https://unsplash.com/es/@alexkixa?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText" Alexandre Debiève
+- "https://unsplash.com/s/photos/technology?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText" Unsplash
+- "https://www.freepik.com/free-photo/remote-meeting-man-working-from-home-during-coronavirus-covid-19-quarantine-remote-office-concept_9367757.htm#query=virtual%20event&position=8&from_view=keyword" Image by master1305 on Freepik
+- "https://www.freepik.com/free-photo/female-team-lead-chatting-with-coworkers-online-meeting-using-videocall-software-laptop-businesspeople-having-discussion-videoconference-telecommunications-remote-teamwork-concept_30770522.htm#query=virtual%20event&position=5&from_view=keyword">Image by DCStudio on Freepik
+  
 ## 📝 License
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [OPTIONAL] Great job adding the MIT file 👍, but I noticed that the MIT link is broken, kindly fix it by changing the path to `[MIT](./MIT.md)` instead of `[MIT](./LICENSE)` 💯